android-components/components/feature/media/src/main/java/mozilla/components/feature/media/service/MediaSessionServiceDelegate.kt

```diff
@@ -180,10 +186,32 @@ internal class MediaSessionServiceDelegate(
     }
 
     @VisibleForTesting
-    internal fun startForeground(sessionState: SessionState) {
-        notificationScope?.launch {
+    @Suppress("TooGenericExceptionCaught")
+    internal fun startForeground(
+        sessionState: SessionState,
+        coroutineContext: CoroutineContext = EmptyCoroutineContext,
```

Comment:
Generally I would agree with the latter route in a perfect world because I think it would increase clarity to downstream consumers, but I am not sure it's worth a breaking change in the API.